config/config_params.go

```diff
@@ -317,6 +317,7 @@ func (config *Config) DatastoreConfig() api.CalicoAPIConfig {
 		if err != nil {
```

Comment:
Yeah, the environment variables all work with the `CALICO_` prefix, or without (with the prefix takes precedence).  Here's an example: http://docs.projectcalico.org/v2.0/reference/calicoctl/setup/kubernetes

Note:

> Note that all environment variables may also be prefixed with “CALICO_”, for example “CALICO_DATASTORE_TYPE” and “CALICO_KUBECONFIG” etc. may also be used. This is useful if the non-prefixed names clash with existing environment variables defined on your system


In `calico/node`, we pass both essentially to work around the same thing you described above: https://github.com/projectcalico/calicoctl/blob/master/calico_node/filesystem/etc/service/available/felix/run#L15